pallets/loans/Cargo.toml

```
@@ -63,6 +64,7 @@ std = [
   "sp-io/std",
   "strum/std",
   "orml-traits/std",
+  "serde/std",
```

Since it's optional, shouldn't we apply `?`
```suggestion
  "serde?/std",
```

---

Good one!